docs/ui/components/utils/infrastructure.tsx

```diff
@@ -1,108 +1,60 @@
-import {
-  HardwareRSpec,
-  HardwareSpecKey,
-  ResourcePlatform,
-  ResourceSpecData,
-} from '~/types/resourceSpecs';
 import { markdownComponents } from '~/ui/components/Markdown';
 
-const {
-  hardware: hardwareSpecs,
-  vm: vmSpecs,
-  resources: { android, ios },
-} = resourceSpecs as ResourceSpecData;
-
-function formatHardware(hardwares: HardwareRSpec) {
-  return keys(hardwares).map(hardware => {
-    const { cpu, memory, description } = hardwareSpecs[hardware];
-    return `${cpu}, ${memory} (${description})`;
-  });
-}
-
-function formatVMs(hardwares: HardwareRSpec) {
-  return entries(hardwares)
-    .map(([hardware, { vm, extra }]) => {
-      const { cpu, memory } = vmSpecs[vm];
-      if (size(hardwares) === 1) return `${cpu}, ${memory}, ${extra}`;
-      return `${cpu}, ${memory}, ${extra} (for builds runnning on ${hardwareSpecs[hardware].name})`;
-    })
-    .join(' or ');
-}
-
-export const iosResourceClasses = values(ios).map(({ symbol }) => symbol);
-export const iosResources = values(ios).map(({ symbol, hardware }) => ({
-  symbol,
-  description: formatVMs(hardware),
-}));
-export const iosHardware = values(ios).flatMap(({ symbol, hardware }) =>
-  keys(hardware).map(name => ({ ...hardwareSpecs[name], symbol }))
-);
-
-export const androidResourceClasses = values(android).map(({ symbol }) => symbol);
-export const androidResources = values(android).map(({ symbol, hardware }) => ({
-  symbol,
-  description: formatHardware(hardware),
-}));
-export const androidHardware = values(android).flatMap(({ symbol, hardware }) =>
-  keys(hardware).map((name: HardwareSpecKey) => ({ ...hardwareSpecs[name], symbol }))
-);
+export const ResourceClasses = { android: ['medium', 'large'], ios: ['medium', 'large'] } as const;
 
-function buildResourceLink(symbol: string, platform: ResourcePlatform) {
-  const platformId = platform === 'ios' ? 2 : 1;
-  return (
+const AndroidResourceClassToSpec: Record<(typeof ResourceClasses.android)[number], JSX.Element> = {
+  medium: (
+    <>
+      4 vCPUs, 16 GB RAM (
+      <markdownComponents.a href="https://cloud.google.com/compute/docs/general-purpose-machines#n2_machine_types">
+        n2-standard-4
+      </markdownComponents.a>{' '}
+      Google Cloud machine type)
+    </>
+  ),
+  large: (
     <>
-      <markdownComponents.code>
-        <markdownComponents.a href={`eas-json/#resourceclass-${platformId}`}>
-          {symbol}
-        </markdownComponents.a>
-      </markdownComponents.code>
-      :
+      8 vCPUs, 32 GB RAM (
+      <markdownComponents.a href="https://cloud.google.com/compute/docs/general-purpose-machines#n2_machine_types">
+        n2-standard-8
+      </markdownComponents.a>{' '}
+      Google Cloud machine type)
     </>
+  ),
+};
+
+const IosResourceClassToSpec: Record<(typeof ResourceClasses.ios)[number], JSX.Element> = {
+  medium: <>3 vCPUs, 8 GB RAM</>,
+  large: <>4 vCPUs, 12 GB RAM (or 22 GB RAM if running on an M2 Mac)</>,
```

Comment:
Should this be 6 vCPUs, and do we need to differentiate between Pro and non-Pro chips?

Reply:
Ah, I haven't merged the pull request bumping to 5/6 yet. Let me do that. When it lands and we see no issues I'll update the pull request and, since then both vCPUs and RAM size will be different for M2 and M2 Pro, I'll separate them clearly.

Reply:
Merged and tested https://github.com/expo/turtle-v2-anka/pull/213. Updated with:

<img width="608" alt="Zrzut ekranu 2023-12-15 o 20 49 24" src="https://github.com/expo/expo/assets/1151041/2a5033fd-1c00-4cd7-9b61-a74582b1cf82">